terrain_navigation_ros has include error for planner_msgs

When building `main` ROS1 branch in docker on hash `4b1ef8e518cc04efb8528147d5a161784c2044cd`, I get the following error:

```
Errors     << terrain_navigation_ros:make /root/catkin_ws/logs/terrain_navigation_ros/build.make.000.log                                                                                                                            
In file included from /root/catkin_ws/src/terrain-navigation/terrain_navigation_ros/src/terrain_planner.cpp:41:
/root/catkin_ws/src/terrain-navigation/terrain_navigation_ros/include/terrain_navigation_ros/terrain_planner.h:67:10: fatal error: planner_msgs/SetPlannerState.h: No such file or directory
   67 | #include <planner_msgs/SetPlannerState.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/terrain_navigation_ros.dir/build.make:63: CMakeFiles/terrain_navigation_ros.dir/src/terrain_planner.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:389: CMakeFiles/terrain_navigation_ros.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
cd /root/catkin_ws/build/terrain_navigation_ros; catkin build --get-env terrain_navigation_ros | catkin env -si  /usr/bin/make --jobserver-auth=3,4; cd -
```

`terrain_navigation_ros/src/terrain_planner.h` is including `planner_msgs/NavigationStatus.h`, but that package isn't linked to. 

I can issue a PR to fix this.

--- a/terrain_navigation_ros/CMakeLists.txt
+++ b/terrain_navigation_ros/CMakeLists.txt
@@ -19,6 +19,7 @@ find_package(catkin REQUIRED COMPONENTS
   grid_map_core
   grid_map_cv
   grid_map_msgs
+  planner_msgs
   grid_map_ros
   grid_map_pcl
   grid_map_geo

